website/content/en/docs/concepts/node-templates.md

```diff
@@ -445,10 +445,6 @@ spec:
 
 For more examples on configuring these fields for different AMI families, see the [examples here](https://github.com/aws/karpenter/blob/main/examples/provisioner/launchtemplates).
 
-{{% alert title="Windows Support Notice" color="warning" %}}
```

Comment:
Docs points to the latest stable version so we should just update the `/preview` for now